src/main/java/duke/logic/sort/SortMealByDefault.java

```diff
@@ -0,0 +1,12 @@
+package duke.logic.sort;
+
+import duke.model.meal.Meal;
+
+import java.util.Comparator;
+
+public class SortMealByDefault implements Comparator<Meal> {
+    @Override
+    public int compare(Meal meal, Meal mealCompared) {
+        return meal.getMealType().compareTo(mealCompared.getMealType());
```

Comment:
I think this will return breakfast, dinner then lunch since B < D < L in alphabetical order? But I think you intended to have B < L < D as mentioned in PR.

Reply:
I have implemented a enum class to handle the ordering of the meal. So the ordering of the meal depends on the ordering in the enum. Since the enum is ordered BREAKFAST, LUNCH, DINNER. The sort will assume this ordering.